conjureup/utils.py

```diff
@@ -512,11 +513,13 @@ def __available_on_darwin(key):
     if is_darwin() and metadata.cloud_whitelist \
        and 'localhost' in metadata.cloud_whitelist:
         return False
+    if metadata.spell_type == consts.spell_types.SNAP:
```

Comment:
This needs an `is_darwin()` check.

Reply:
Tested on Ubuntu, and microk8s no longer shows up at all.

Reply:
Or you could refactor this function and do the `is_darwin()` check before any of the others, so that you only have to do it once.

Reply:
To be fair, i was only responsible for macos testing ;)